docker/web.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
+
+RUN npm install concurrently -g
+
+WORKDIR /app
+
+RUN git clone https://github.com/juspay/hyperswitch-web .
+
+RUN sed -i '/hot: true,/a \  host: "0.0.0.0",' webpack.dev.js
+RUN sed -i '/hot: true,/a \  host: "0.0.0.0",' Hyperswitch-React-Demo-App/webpack.dev.js
```

Comment:
might not be needed since after [this](https://github.com/juspay/hyperswitch-web/pull/294).